db/version_edit.h

```diff
@@ -660,6 +673,22 @@ class VersionEdit {
 
   const char* DecodeNewFile4From(Slice* input);
 
+  void EncodeFileBoundaries(std::string* dst, const FileMetaData& meta,
```

Comment:
I don't think this implementation belongs in the header, and could use a short comment

Reply:
I moved the implementation to the source file and added a comment. Thanks for the suggestion!